.github/workflows/mac.yml

```diff
@@ -482,17 +546,19 @@ jobs:
           # Find and install OV dev wheel
           pushd ${{ env.INSTALL_DIR }}/tools
             ov_dev_wheel_name=$(find . -name 'openvino_dev*.whl')
-            python3 -m pip install $ov_dev_wheel_name[mxnet,caffe,kaldi,onnx,tensorflow2]
```

Comment:
note, that mxnet is available only for macos x86_64
explicitly removed this extras in https://github.com/openvinotoolkit/openvino/pull/20403

So, we need conditional install depending on architecture (and similar for tests below)